lib/zone-spec/proxy.ts

```diff
@@ -0,0 +1,128 @@
+(function () {
+  class ProxyZoneSpec implements ZoneSpec {
+    name: string = 'ProxyZone';
+
+    private _delegateSpec: ZoneSpec;
+
+    properties: {[k: string]: any} = {'ProxyZoneSpec': this};
```

Comment:
Ah no, I see that it returns itself. Probably doesn't make sense to change it then.